packages/block-editor/src/components/block-list/index.js

```diff
@@ -41,13 +46,55 @@ import {
 	DEFAULT_BLOCK_EDIT_CONTEXT,
 } from '../block-edit/context';
 
-const elementContext = createContext();
+const componentsContext = createContext();
```

Comment:
Should it be `ComponentsContext`?

This also confused me when I first noticed `elementsContext`, all the other contexts in the codebase have an uppercase first character.

Reply:
It's not a component? :)